README.md

```diff
@@ -1,44 +1,12 @@
-# Submariner Operator
+# subctl
 
 <!-- markdownlint-disable line-length -->
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/4865/badge)](https://bestpractices.coreinfrastructure.org/projects/4865)
-[![Release Images](https://github.com/submariner-io/submariner-operator/workflows/Release%20Images/badge.svg)](https://github.com/submariner-io/submariner-operator/actions?query=workflow%3A%22Release+Images%22)
-[![Periodic](https://github.com/submariner-io/submariner-operator/workflows/Periodic/badge.svg)](https://github.com/submariner-io/submariner-operator/actions?query=workflow%3APeriodic)
-[![Flake Finder](https://github.com/submariner-io/submariner-operator/workflows/Flake%20Finder/badge.svg)](https://github.com/submariner-io/submariner-operator/actions?query=workflow%3A%22Flake+Finder%22)
+[![Release Images](https://github.com/submariner-io/subctl/actions/workflows/release_subctl_on_push.yml/badge.svg)](https://github.com/submariner-io/subctl/actions/workflows/release_subctl_on_push.yml)
+[![Periodic](https://github.com/submariner-io/subctl/actions/workflows/periodic.yml/badge.svg)](https://github.com/submariner-io/subctl/actions/workflows/periodic.yml)
 <!-- markdownlint-enable line-length -->
 
-The Submariner Operator installs the Submariner components on a Kubernetes cluster. It's also available on
-[OperatorHub](https://operatorhub.io/operator/submariner).
+The `subctl` command-line utility simplifies the deployment and maintenance of Submariner by automating interactions with the [Submariner
+Operator](https://github.com/submariner-io/submariner-operator) and providing diagnostic features.
 
-## Quickstart
-
-Please refer the Quickstart Guides on Submariner's website:
-
-* [Sandbox Environment (kind)](https://submariner.io/getting-started/quickstart/kind/).
-* [OpenShift on AWS](https://submariner.io/getting-started/quickstart/openshift/aws/).
-* [OpenShift on AWS with Globalnet](https://submariner.io/getting-started/quickstart/openshift/globalnet/).
-* [OpenShift on AWS and vSphere](https://submariner.io/getting-started/quickstart/openshift/vsphere-aws/).
-* [Google Kubernetes Engine](https://submariner.io/getting-started/quickstart/managed-kubernetes/gke/).
-* [Rancher](https://submariner.io/getting-started/quickstart/managed-kubernetes/rancher/).
-
-## Subctl Releases
```

Comment:
This portion needs to stay. Its about `subctl` releases. The links would need to be changed.